include/SFML/Window/Keyboard.hpp

```diff
@@ -171,12 +345,73 @@ class SFML_WINDOW_API Keyboard
     ////////////////////////////////////////////////////////////
     static bool isKeyPressed(Key key);
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Check if a key is pressed
+    ///
+    /// \param code Scancode to check
+    ///
+    /// \return True if the physical key is pressed, false otherwise
+    ///
+    ////////////////////////////////////////////////////////////
+    static bool isKeyPressed(Scancode code);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Localize a physical key to a logical one
+    ///
+    /// \param code Scancode to localize
+    ///
+    /// \return The key corresponding to the scancode under the current
+    ///         keyboard layout used by the operating system, or
+    ///         sf::Keyboard::Unknown when the scancode cannot be mapped
+    ///         to a Key.
+    ///
+    /// \see unlocalize
+    ///
+    ////////////////////////////////////////////////////////////
+    static Key localize(Scancode code);
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Identify the physical key corresponding to a logical one
+    ///
+    /// \param key Key to "unlocalize"
+    ///
+    /// \return The scancode corresponding to the key under the current
+    ///         keyboard layout used by the operating system, or
+    ///         sf::Keyboard::ScanUnknown when the key cannot be mapped
+    ///         to a sf::Keyboard::Scancode.
+    ///
+    /// \see localize
+    ///
+    ////////////////////////////////////////////////////////////
+    static Scancode unlocalize(Key key);
```

Comment:
Not being familiar with this domain, is "unlocalize" a common term? What about "delocalize"?
I'm fine with either 🙂

Reply:
If online word directories are any indication, then delocalize seems to be a verb and unlocalize doesn't seem to be one.

- https://www.merriam-webster.com/dictionary/delocalize
- https://www.merriam-webster.com/dictionary/unlocalize
- https://www.thefreedictionary.com/delocalize
- https://www.thefreedictionary.com/unlocalize

Yet "delocalized" is an accepted word again, but it's more about the location and not the locality.

- https://www.merriam-webster.com/dictionary/unlocalized
- https://www.thefreedictionary.com/unlocalized

Then again, technical words don't always have to be part of a dictionary to be utilized. Like you won't generally find "performant" in a dictionary, yet it's a word most developers will be using naturally.

Unless someone else has an objection, I'd also go with the more "matching" with the English language "delocalize".